submissions/Vitaminvp/HTML&CSSPractice/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Responsive Side Menu</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<header>
+  <nav class="nav">
+      <input type="checkbox" class="nav__input" id="nav__input">
+      <label for="nav__input">
```

Comment:
You used `label` as a `button`, so it would be nice to use `role` attribute here. See more about [WAI ARIA](https://www.w3.org/WAI/standards-guidelines/aria/).
It will make your website more accessible for screen readers and more useful for people with disabilities.
I recommend you to watch Vadim Makeev's videos about semantics and accessibility. But this is up to you :)

Reply:
Add role="button"